### PR TITLE
Update deadline exception messages

### PR DESCRIPTION
**Pull request by dlimyy**
Updated deadline exception messages and modified
command format for message usages for all tags.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/DeleteTagCommand.java [hunk 1]
```diff
@@ -19,9 +19,8 @@
 public class DeleteTagCommand extends Command {
     public static final String COMMAND_WORD = "tagdel";
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": deletes tags linked to the task.\n"
-            + "Format: " + COMMAND_WORD + " " + "[" + PREFIX_TAG + "TAG]\n"
-            + "Example: " + COMMAND_WORD + " " + "t/" + "priority\n"
-            + COMMAND_WORD + " " + PREFIX_TAG + "priority deadline";
+            + "Parameters: " + COMMAND_WORD + " " + "INDEX " + PREFIX_TAG + "KEYWORD [SECOND_KEYWORD]\n"
+            + "Example: " + COMMAND_WORD + " " + "t/" + "priority\n";
```
Comment (RachelChua) [inline src/main/java/seedu/address/logic/commands/DeleteTagCommand.java:23]: example should be with a t infront of the command word

**Review by RachelChua — CHANGES_REQUESTED** [review]
LGTM other than the part for the example in message usage.

File: src/main/java/seedu/address/logic/commands/DeleteTagCommand.java (hunk 1 above)
Comment (dlimyy) [reply src/main/java/seedu/address/logic/commands/DeleteTagCommand.java:23]: yup i just changed all of the message usages

Comment (RachelChua) [reply src/main/java/seedu/address/logic/commands/DeleteTagCommand.java:23]: wait i mean in ur example

Comment (dlimyy) [reply src/main/java/seedu/address/logic/commands/DeleteTagCommand.java]: wait sorry I have just noticed it. I have included it in.

**Review by RachelChua — APPROVED** [review]
LGTM!

**Pull request merged**